spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+
+        <emu-clause id="sec-temporal-calendardatefromfields">
+          <h1>CalendarDateToISO ( _calendar_, _fields_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateToISO performs implementation-defined processing to convert _fields_, which is a Record that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateaddition">
+          <h1>CalendarDateAddition ( _calendar_, _date_, _duration_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateAddition performs implementation-defined processing to add _duration_ to _date_ in the context of the calendar represented by the string _calendar_ and returns the corresponding day, month and year of the result in the ISO calendar values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference">
+          <h1>CalendarDateDifference ( _calendar_, _one_, _two_, _largestUnit_ )</h1>
+          <p>The abstract operation CalendarDateDifference performs implementation-defined processing to find the difference between the two dates _one_ and _two_ in the context of the calendar represented by the string _calendar_ and returns the corresponding years, months, weeks and days as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateera">
+          <h1>CalendarDateEra ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEra performs implementation-defined processing to find the era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateerayear">
+          <h1>CalendarDateEraYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateEraYear performs implementation-defined processing to find the year of the current era for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateyear">
+          <h1>CalendarDateYear ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateYear performs implementation-defined processing to find the year for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonth">
+          <h1>CalendarDateMonth ( _calendar_, _date_ )</h1>
+          <p>The abstract operation CalendarDateMonth performs implementation-defined processing to find the month for the date corresponding to _date_ in the context of the calendar represented by the string _calendar_ and returns it.</p>
+        </emu-clause>
```

Comment:
Should clarify that this should be the ordinal index of the month in the current year. For example, Adar II in a Hebrew leap year (month code `M05L`) is the sixth month so `6` will be returned.

Reply:
I still think a note is justified here, e.g. _The return value represents the ordinal index of the month in the current year. For lunisolar calendars, the same month can return a different index depending on whether it's in a leap year._